<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE ProvisioningTransaction PUBLIC "sailpoint.dtd" "sailpoint.dtd">
<ProvisioningTransaction accountDisplayName="CN=elomus1,OU=People,OU=Demo,DC=Eshiam,DC=com" applicationName="Active- Directory" identityDisplayName="elomus1" identityName="elomus1" integration="Active- Directory" name="0000000150" nativeIdentity="CN=elomus1,OU=People,OU=Demo,DC=Eshiam,DC=com" operation="Create" source="RapidSetup" status="Failed" type="Auto">
  <Attributes>
    <Map>
      <entry key="accessRequestId" value="0000000471"/>
      <entry key="request">
        <value>
          <AccountRequest application="Active- Directory" assignmentIds="faecc182bb8848aeb4ea1263da1ec1b4" nativeIdentity="CN=elomus1,OU=People,OU=Demo,DC=Eshiam,DC=com" op="Create">
            <Attributes>
              <Map>
                <entry key="provisioningTransactionId" value="c0a80103858c19a881859b13eafa0746"/>
              </Map>
            </Attributes>
            <AttributeRequest name="memberOf" op="Add" value="CN=All_Users,OU=Groups,OU=Demo,DC=Eshiam,DC=com"/>
            <AttributeRequest name="objectType" op="Set" value="User"/>
            <AttributeRequest name="sAMAccountName" op="Set" value="Musk"/>
            <AttributeRequest name="password" op="Set">
              <Attributes>
                <Map>
                  <entry key="secret" value="true"/>
                </Map>
              </Attributes>
            </AttributeRequest>
            <AttributeRequest name="givenName" op="Set" value="Elon"/>
            <AttributeRequest name="sn" op="Set" value="Musk"/>
            <AttributeRequest name="employeeId" op="Set" value="1a2p3k"/>
            <AttributeRequest name="homeDirectory" op="Set" value="C:\U105"/>
            <AttributeRequest name="homeDrive" op="Set" value="C:"/>
            <AttributeRequest name="msNPAllowDialin" op="Set" value="Not Set"/>
            <ProvisioningResult status="failed">
              <Errors>
                <Message key="Exception occurred while executing the RPCRequest: Errors returned from IQService. &quot;The object already exists. 00002071: UpdErr: DSID-03050371, problem 6005 (ENTRY_EXISTS), data 0 . HRESULT:[0x80071392] For identity: CN=elomus1,OU=People,OU=Demo,DC=Eshiam,DC=com&quot;" type="Error"/>
              </Errors>
            </ProvisioningResult>
          </AccountRequest>
        </value>
      </entry>
    </Map>
  </Attributes>
</ProvisioningTransaction>